codespell_lib/data/dictionary.txt

```diff
@@ -14105,6 +14107,7 @@ tanget->tangent
 tangetial->tangential
 tangetially->tangentially
 tangets->tangents
+tanks->thanks
```

Comment:
Plural of e.g. storage tank or battle tank

Reply:
removed